MapboxNavigation/MGLStyle.swift

```diff
@@ -9,10 +9,10 @@ extension MGLStyle {
     static let mapboxChinaNightStyleURL = URL(string:"mapbox://styles/mapbox/dark-zh-v1")!
     
     // Default Mapbox Navigation Day Style URL.
-    static let defaultNavigationDayStyleURL = URL(string:"mapbox://styles/mapbox-map-design/ckd6dqf981hi71iqlyn3e896y")!
+    static let defaultNavigationDayStyleURL = URL(string:"mapbox://styles/mapbox/navigation-day-v1")!
     
     // Default Mapbox Navigation Night Style URL.
-    static let defaultNavigationNightStyleURL = URL(string:"mapbox://styles/mapbox-map-design/ckd6dnz2q0m0q1io1mssumxqd")!
+    static let defaultNavigationNightStyleURL = URL(string:"mapbox://styles/mapbox/navigation-night-v1")!
```

Comment:
Since this is technically v1 all over again, how do you feel about changing `navigationDayStyleURL(version:)` to return `mapbox/navigation-day-v\(version)` and removing the ability to get the Navigation Guidance and Navigation Preview styles altogether via this extension? If a developer really needs either style, they can construct its URL directly.

Reply:
Seems like a good idea, on the other hand it might not be clear how to construct such URLs for end user. I've mentioned how to do that in comments to `navigationDayStyleURL(version:)` and `navigationNightStyleURL(version:)` methods.